src/client/datascience/interactive-window/interactiveWindow.ts

```diff
@@ -535,27 +528,32 @@ export class InteractiveWindow extends InteractiveBase implements IInteractiveWi
             return this.extensionChecker.showPythonExtensionInstallRequiredPrompt();
         }
 
-        let model: INotebookModel;
-        this.startProgress();
-        try {
-            model = await this.exportUtil.getModelFromCells(cells);
-        } finally {
-            this.stopProgress();
+        // Pull out the metadata from our active notebook
+        const metadata: nbformat.INotebookMetadata = { orig_nbformat: 3 };
```

Comment:
This wasn't working before. Now the exported notebook will have the actual kernel inside of it.